packages/pdfgrep/build.sh

```diff
@@ -7,3 +7,8 @@ TERMUX_PKG_REVISION=7
 TERMUX_PKG_SRCURL=https://pdfgrep.org/download/pdfgrep-$TERMUX_PKG_VERSION.tar.gz
 TERMUX_PKG_SHA256=0ef3dca1d749323f08112ffe68e6f4eb7bc25f56f90a2e933db477261b082aba
 TERMUX_PKG_DEPENDS="libc++, libgcrypt, libgpg-error, pcre, poppler"
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_pre_configure() {
+	autoreconf -i -I$TERMUX_PREFIX/share/aclocal
```

Comment:
Package compiles fine without this change, so wondering why it is necessary there?
```
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/home/builder/.termux-build/pdfgrep/build/testsuite/pdfgrep.tests'
make[2]: Leaving directory '/home/builder/.termux-build/pdfgrep/build/testsuite/pdfgrep.tests'
make[2]: Entering directory '/home/builder/.termux-build/pdfgrep/build/testsuite'
make[3]: Entering directory '/home/builder/.termux-build/pdfgrep/build/testsuite'
make[3]: Nothing to be done for 'install-exec-am'.
make[3]: Nothing to be done for 'install-data-am'.
make[3]: Leaving directory '/home/builder/.termux-build/pdfgrep/build/testsuite'
make[2]: Leaving directory '/home/builder/.termux-build/pdfgrep/build/testsuite'
make[1]: Leaving directory '/home/builder/.termux-build/pdfgrep/build/testsuite'
make[1]: Entering directory '/home/builder/.termux-build/pdfgrep/build'
make[2]: Entering directory '/home/builder/.termux-build/pdfgrep/build'
make[2]: Nothing to be done for 'install-exec-am'.
make[2]: Nothing to be done for 'install-data-am'.
make[2]: Leaving directory '/home/builder/.termux-build/pdfgrep/build'
make[1]: Leaving directory '/home/builder/.termux-build/pdfgrep/build'
termux - build of 'pdfgrep' done
builder@e6cf486fc963:~/termux-packages$
```